linux_new/jdk/debian/src/main/packaging/temurin/8/debian/postinst.in

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+set -eu
+
+priority="@priority@"
+jdk_base_dir="/usr/lib/jvm/@jvm_dir@"
+tools="@jvm_tools@"
+
+case "$1" in
+configure)
+	for tool in $tools; do
+		for tool_path in "$jdk_base_dir/bin/$tool" "$jdk_base_dir/lib/$tool"; do
+			if [ ! -e "$tool_path" ]; then
+				continue
+			fi
+
+			slave=""
```

Comment:
Perhaps "slave" variable name could be changed? In an ideal world we'd have an alternate option for the "--slave" as well, but I'm not seeing one right now, so we may have to put up with it for now.

Reply:
Sadly this is one we will have to put up with, its a feature of the debian installer build.